internal/show/all.go

```diff
@@ -0,0 +1,63 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
Why wasn't this file moved instead of copied?

Reply:
It was copied. See commit https://github.com/submariner-io/submariner-operator/pull/2006/commits/f897e630eb10e28832946f97fa26c902092388b5. Not sure what happened here.